### PR TITLE
pdfcrack: init 0.16

### PR DESCRIPTION
**Pull request by qoelet**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/security/pdfcrack/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  version = "0.16";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/pdfcrack/pdfcrack/pdfcrack-${version}.tar.gz";
+    sha256 = "1vvkrg3niinz0j9wwm31laxgmd7wdz201kn82b3dbksc0w1v4rbq";
+  };
+
+  installPhase = ''
+    mkdir -p $out/bin
+    mv pdfcrack $out/bin
```
Comment (dotlambda) [inline pkgs/tools/security/pdfcrack/default.nix]: `install -Dt $out/bin pdfcrack`

File: pkgs/tools/security/pdfcrack/default.nix [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  '';
+
+  meta = with lib; {
+    homepage = http://pdfcrack.sourceforge.net/;
+    description = "Small command line driven tool for recovering passwords and content from PDF files";
+    license = with licenses; [ gpl2 ];
+    platforms = stdenv.lib.platforms.linux;
```
Comment (dotlambda) [inline pkgs/tools/security/pdfcrack/default.nix]: no `stdenv.lib`

File: pkgs/tools/security/pdfcrack/default.nix (hunk 1 above)
Comment (qoelet) [reply pkgs/tools/security/pdfcrack/default.nix]: That's definitely nicer, amended.

**Comment by dotlambda** [comment]
@GrahamcOfBorg build pdfcrack

**Pull request merged**